tools/flang2/flang2exe/cgmain.cpp

```diff
@@ -13260,11 +13260,16 @@ print_function_signature(int func_sptr, const char *fn_name, LL_ABI_Info *abi,
   if (need_debug_info(SPTR_NULL)) {
     /* 'attributes #0 = { ... }' to be emitted later */
     print_token(" #0");
-  } else if (!XBIT(183, 0x10)) {
+  } else if (!XBIT(183, 0x10) || XBIT(14, 0x8)) {
```

Comment:
I assuming this is already documented ? this `NOINLINE` bit you're using :)

Reply:
Yes it is
> Don't extract this function (set by pragma noinline)

:)